src/geotiffwriter.js

```diff
@@ -352,7 +387,15 @@ export function writeGeotiff(data, metadata) {
 
   if (!metadata.StripByteCounts) {
     // we are only writing one strip
-    metadata.StripByteCounts = [numBands * height * width];
+
+    // default for Uint8
+    let elementSize = 1;
```

Comment:
I'm wondering if we want to change the default to 8 for Float64 instead of Uint8?  Ideally, the writer will work for the greatest number of use cases by default.  If someone wants Uint8, they could pass in a Uint8Array of values instead of an Array.